src/tvheadend/InstanceSettings.cpp

```diff
@@ -188,6 +193,8 @@ ADDON_STATUS InstanceSettings::SetSetting(const std::string& key,
     return SetStringSetting(GetStreamingProfile(), value);
   else if (key == "streaming_http")
     return SetBoolSetting(GetStreamingHTTP(), value);
+  else if (key == "streaming_stalledtreshold")
```

Comment:
To be consistent with the code the setting should probably be named stream_stalled_threshold

Reply:
Will change that.